rollup-sequencer/src/utils/index.ts

```diff
@@ -298,6 +288,34 @@ function print(data: any) {
 	console.log(util.inspect(data, { depth: null }));
 }
 
+async function getUpdateForL2(publicClient: any) {
```

Comment:
Add this type to this import **type PublicClient**
import { type PublicClient, encodeAbiParameters, keccak256 } from "viem";

and then in the function replace any with **PublicClient**